RSDCrap/Classes/All.uc

```diff
@@ -0,0 +1,23 @@
+//=============================================================================
+// AllUI.
+//=============================================================================
+class All expands Object
```

Comment:
Giving this a better name might make sense. Even like `class RSDCrap`.

Reply:
Probably, I just went by the standard name definition for everything (AllUi, AllTexture, etc)

Reply:
Nope, class name can't be the same as the package name